QUEST.tsv

```diff
@@ -2272,24 +2272,24 @@ QUEST_20150717_002271	[Usage Limit]{nl}- Due to a recent incident, we sealed the
 QUEST_20150717_002272	[Method of Usage]{nl}1. Use the Schilt Essences to the portal generator.{nl}2. One person could enter when a certain number of Essences are used.{nl}3. When more than one person enter simultaneously, please ask the chief.
 QUEST_20150717_002273	The essence of spark forms here.{nl}You may get burned so be careful.
 QUEST_20150717_002274	Modestas accepted his defeat and promised to pay back later. When Modestas was about to leave with his horse and wagon, the old Alchemist with a grin Modestas
-QUEST_20150717_002275
-QUEST_20150717_002276
+QUEST_20150717_002275 Story of the Flame/Fire
+QUEST_20150717_002276 
 QUEST_20150717_002277
-QUEST_20150729_002281
-QUEST_20150729_002282
-QUEST_20150729_002283
-QUEST_20150729_002284
-QUEST_20150729_002285
-QUEST_20150729_002286
-QUEST_20150729_002287
-QUEST_20150729_002288
-QUEST_20150729_002289
-QUEST_20150729_002290
-QUEST_20150729_002291
-QUEST_20150729_002292
-QUEST_20150729_002293
-QUEST_20150729_002294
-QUEST_20150729_002295
+QUEST_20150729_002281 Hm.. Have the party members fully gathered yet?
+QUEST_20150729_002282 Welcome to the Mercenary Quest Office. {nl}But.. There aren't appropriate requests yet.{nl} 
+QUEST_20150729_002283 When you return with reliable comrades, when I will give you an appropriate quest.{nl}If there is a request.{nl} 
+QUEST_20150729_002284 There is a good request that just came in.{nl}Do you want to take it?
+QUEST_20150729_002285 Anyway, that is very strange.{nl}If it was Goddess Laima's will, not anyone could undo her seal.
+QUEST_20150729_002286 Whether this revelation will become a new beacon of hope or a start of a new disaster, it is indeed difficult to judge.. {nl}We believe we will continue to investigate. 
+QUEST_20150729_002287 )}My son volunteered to join the army.{nl}So I have been offering my prayers to the greatest Goddess Ausrine.{nl}
+QUEST_20150729_002288 )}Monsters are keep multiplying in number to the point the people cannot safely live in the forest anymore.{nl}At least Klaipeda is still safe.{nl}
+QUEST_20150729_002289 Maybe the Goddess Ausrine gave us her protection.{nl}Isn't she the Goddess who leads all other Goddesses?
+QUEST_20150729_002290 )}I am glad my son dreams to become a soldier..{nl}But the monsters are keep increasing in number, I am just worried he will be wounded.
+QUEST_20150729_002291 
+QUEST_20150729_002292 After the night passes, doesn't the morning dawn? The Goddess Ausrine is the one who brings us the morn. {nl}Not only that, but when we die we go to the Goddess Ausrine.
```

Comment:
"Doesn't the morning come after the night? ...the one who brings us the morning."